node-it/src/test/scala/com/wavesplatform/it/asset/IssueReissueBurnAssetSuite.scala

```diff
@@ -338,71 +391,256 @@ class IssueReissueBurnAssetSuite extends BaseSuite {
     tx._1
   }
 
-  def validateIssuedAssets(account: String, tx: Transaction, data: Asset, nth: Int = -1, method: String): String = {
-    val assetId = method match {
-      case "@Callable" =>
-        (if (nth == -1) sender.debugStateChanges(tx.id.toString).stateChanges.get.issues.head
-         else sender.debugStateChanges(tx.id.toString).stateChanges.get.issues(nth)).assetId
-      case _ => tx.id
+  def assertStateChanges(tx: Transaction)(f: StateChangesDetails => Unit): Unit = {
+    f(stateChanges(tx))
+    f(stateChangesStrings(tx))
+    f(grpcStateChanges(tx))
```

Comment:
This check is redundant as grpc stateChanges's already checked in GrpcIssueReissueBurnAssetSuite. Delete it.

Reply:
*line 397